mock/api.js

```diff
@@ -150,42 +133,46 @@ export const mockSamples = {
   },
 };
 
-export const mockDetail = {
-  hits: {
-    hits: [
-      {
-        _source: {
-          run: {
-            id: 'test_id',
-            controller: 'test_controller',
-            name: 'test_result',
-            script: casual.random_element(['fio', 'uperf']),
-            date: moment.utc(),
-            start: moment.utc(),
-            end: moment.utc(),
-            user: casual.username,
-          },
-          host_tools_info: [
-            {
-              hostname: 'test_host',
-              tools: {
-                iostat: '--interval=3',
-                mpstat: '--interval=3',
-                perf: "--record-opts='record -a --freq=100'",
-                pidstat: '--interval=30',
-                'proc-interrupts': '--interval=3',
-                'proc-vmstat': '--interval=3',
-                sar: '--interval=3',
-                turbostat: '--interval=3',
-              },
-            },
-          ],
-        },
-      },
-    ],
+export const mockDetail = hostname => ({
+  hostTools: new Array(DEFAULT_SIZE).fill().map(() => ({
+    hostname: hostname.split('.')[0],
+    'hostname-f': hostname,
+    tools: {
+      'hostname-alias': '',
+      'hostname-all-fqdns': hostname,
+      'hostname-all-ip-addresses': casual.ip,
+      'hostname-domain': `${casual.word}.${casual.word}.com`,
+      'hostname-fqdn': hostname,
+      'hostname-ip-address': casual.ip,
+      'hostname-nis': 'hostname: Local domain name not set',
+      'hostname-short': hostname.split('.')[0],
+      'rpm-version': `v0.+${casual.integer}`,
+      tools: casual.word,
+      vmstat: '',
+    },
+  })),
+  runMetadata: {
+    config: casual.word,
+    controller: hostname,
+    controller_dir: 'gprfc056.sbu.lab.eng.bos.redhat.com',
+    date: moment.utc(),
+    end: moment.utc(),
+    start: moment.utc(),
+    'file-date': moment.utc(),
+    'file-name': `/${casual.work}/${casual.work}/${casual.work}`,
+    'file-size': casual.building_number,
+    id: casual.uuid.replaceAll('-', ''),
```

Comment:
I assume some test environments will run into the same old browser version Gurbir hit with `replaceAll`? Should they all be changed "to be safe"???

Reply:
I have replaced it with `casual.uuid.replace(/-/g, '');`. Hopefully it will work now.